flecsi/coloring/dcrs_utils.h

```diff
@@ -436,7 +437,7 @@ make_dcrs_distributed(
 
   size_t max_global_vert_id{0};
   for(auto v : vertex_local_to_global)
-    max_global_vert_id = std::max(max_global_vert_id, v);
+    max_global_vert_id = (std::max)(max_global_vert_id, v);
```

Comment:
The Windows headers still define min()/max() as macros. One can disable those macros by adding `-DNOMINMAX` to the compilers command line. This is certainly an option while compiling FLeCSI itself, however might interfere with dependent projects. Please advise how to proceed. I personally think that the additional parenthesis' are the least invasive solution, but I'm happy do adapt the code to your guidelines.

Reply:
I think I agree that the parentheses, as odd as they look, are the cleanest solution for a library that shouldn't be requiring clients to globally configure their preprocessor.  I just wanted to make sure that we weren't doing so because it was _once_ necessary.